pyproject.toml

```diff
@@ -25,6 +25,10 @@ add_job = "jobrunner.cli.add_job:run"
 kill_job = "jobrunner.cli.kill_job:run"
 local_run = "jobrunner.cli.local_run:run"
 retry_job = "jobrunner.cli.retry_job:run"
+prepare_for_reboot = "jobrunner.cli.prepare_for_reboot:run"
+flags = "jobrunner.cli.flags:run"
+migrate = "jobrunner.cli.migrate:run"
```

Comment:
Could you alphabetise these? Makes them a little easier to skim 🙂

Reply:
done